test/test_hydrostatic_free_surface_immersed_boundaries_congrad_solve.jl

```diff
@@ -1,5 +1,3 @@
-include("dependencies_for_runtests.jl")
-
```

Comment:
Let's keep this so that the tests can be run standalone via

```julia
julia> include("test_hydrostatic_free_surface_immersed_boundaries_congrad_solve.jl")
```

Super handy for development esp when using Revise to avoid waiting for recompilation.